Maven Central Deployer URL note

[maven-central.adoc](https://github.com/jreleaser/jreleaser.github.io/blob/main/docs/modules/examples/pages/maven/maven-central.adoc) defines: 

```
:deployer_url:  https://s01.oss.sonatype.org/service/local
```

However for Sonatype projects created before February 2021 [should use](https://central.sonatype.org/publish/release/#login-into-ossrh) `https://oss.sonatype.org/service/local`. Otherwise, JReleaser fails with following error - 

```
[ERROR] JReleaser failed after 2.703 s
Unexpected error
Could not find a staging profile matching com.github.manikmagar
```
Root cause isn't very obvious from the error but it is due to the wrong URL being used (if followed the docs).  Maybe adding a note there could help.

--- a/docs/modules/examples/pages/maven/maven-central.adoc
+++ b/docs/modules/examples/pages/maven/maven-central.adoc
@@ -1,5 +1,6 @@
 = Publishing to Maven Central
 :deployer_url:  https://s01.oss.sonatype.org/service/local
+:deployer_old_url:  https://oss.sonatype.org/service/local
 
 Maven Central is the defacto artifact repository for JVM based projects. Anyone can publish artifacts to it as long as
 they follow the rules. Follow this link:https://central.sonatype.org/publish/publish-guide/[guide] to register an account.
@@ -21,6 +22,8 @@ performs the following tasks:
 You have the option to close and release the staged repository automatically right after deployment, or keep the staged
 repository open and perform close and release operations using the UI. You must login into Sonatype OSSRH using your
 Sonatype account to do so.
+
+CAUTION: As described https://central.sonatype.org/publish/release/#login-into-ossrh[here], Sonatype projects created before February 2021 may need to use "{deployer_old_url}" instead of "{deployer_url}". Using incorrect url may cause 'Could not find a staging profile matching ...' error.
 
 [tabs]
 ====

